codewars8.py

```diff
@@ -0,0 +1,108 @@
+# https://www.codewars.com/kata/58b3c2bd917a5caec0000017/train/python
+# I know the code doesn't  work as it should... yet. I will fix it later.
+# def sum_groups(arr):
+
+arr = [2, 1, 2, 2, 6, 5, 0, 2, 0, 5, 5, 7, 7, 4, 3, 3, 9]
+
+for index, x in enumerate(arr):
+    if arr[x] % 2 == 0:
```

Comment:
Hi Piotr, it looks like your logic in codewars8 is fine. From a quick look at the code, I think that in the if statement blocks you use arr[x], while you want to use arr[index]. Arr[index] allow you to go through the list step by step, arr[x] will jump randomly through the list, depending what the value of the x is.

Don't be surprised if you encounter `IndexError: list index out of range`, it's because for the last index, you will not be able to find index+1 value to compare as it simply doesn't exist. You might need to create a workaround on top of your code for checking the last index.